csrc/executor_utils.cpp

```diff
@@ -1240,7 +1271,7 @@ std::tuple<NvrtcFunction, std::string, std::vector<char>> nvrtcCompile(
         }
         std::cout << log.data() << std::endl;
       }
-      AT_CUDA_NVRTC_CHECK(result);
+      NVRTC_SAFE_CALL(result);
```

Comment:
Is there any specific reason to do this check here? There's also a check at line 1262.

Reply:
No. I think we should have replaced 1262 with this one.

Reply:
We should probably just this line. Line 1262 doesn't just assert on the return value, but it also gives the "log" of the compilation.